game/src/devtools/fifteen_min/mod.rs

```diff
@@ -24,6 +24,7 @@ mod isochrone;
 /// This is the UI state for exploring the isochrone/walkshed from a single building.
 pub struct Viewer {
     panel: Panel,
+    start_building_id: BuildingID,
```

Comment:
`self.isochrone.start` has it; I was trying to keep "raw data" organized there, and only have UI state here